plugins/modules/cloudflare_dns.py

```diff
@@ -158,7 +158,7 @@
       - Support for V(SPF) has been removed from community.general 9.0.0 since that record type is no longer supported by
```

Comment:
```suggestion
      - Support for V(SPF) has been removed from community.general 9.0.0 since that record type is no longer supported by
      - Support for V(PTR) has been added in community.general 11.1.0.
```